app.js

```diff
@@ -0,0 +1,113 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const app = express()
+
+
+const siswa=[
+    {
+        id: 1,
+        name: 'wirahadi',
+        role: 'student'
+    },
+    {
+        id:2,
+        name: 'ihsan',
+        role: 'student'
+    },
+    {
+        id:3,
+        name: 'iqbal',
+        role: 'student'
+    }
+];
+
+const mentor=[
+    {
+        id: 1,
+        name:'mas eddy',
+        batch: 'fantastic falkour'
+    },
+    {
+        id: 2,
+        name:'mas afis',
+        batch:'fantastic falkour'
+    },
+    {
+        id:3,
+        name: 'mas luthfi',
+        batch: 'fantastic falkour'
+    }
+]
+
+const course=[
+    {
+        id: 1,
+        namaCourse: 'javascript',
+        tingkatKesulitan: 'sangat susah'
+    },
+    {
+        id:2,
+        namaCourse: 'html',
+        tingkatKesulitan: 'menengah',
+    },
+    {
+        id:3,
+        namaCourse: 'reactjs',
+        tingkatKesulitan: 'menengah',
+    }
+];
+
+app.use(express.urlencoded({ extended: true }));
+
+app.get('/', function(req, res){
+    res.status(200).json(siswa);
+});
+
+app.get('/siswa/:siswaId', (req, res) => {
+    res.status(200).json(siswa.filter((siswa)=> siswa.id === +req.params.siswaId));
```

Comment:
kalau gini return datanya berupa array of object kan ?

untuk detail sepertinya output jangan berupa array lagi, namun langsung ke objectnya